web/src/narrow.js

```diff
@@ -1028,6 +1028,16 @@ export function deactivate(coming_from_recent_topics = false, is_actively_scroll
     if (narrow_state.filter() === undefined && !coming_from_recent_topics) {
         return;
     }
+    // If we're coming from recent topics to All messages, we need to hide
+    // the recent topics view.
```

Comment:
It'd be great to use "Recent conversations" in these comments and to update the one above as well. We haven't done a sweep of these yet since the rename of the view, but it would be good to not add new ones.

Reply:
I did a sweep to rename existing references! #26651